android-project/app/src/main/java/org/diasurgical/devilutionx/DataActivity.java

```diff
@@ -33,6 +34,12 @@ protected void onCreate(Bundle savedInstanceState) {
 
 		((TextView) findViewById(R.id.full_guide)).setMovementMethod(LinkMovementMethod.getInstance());
 		((TextView) findViewById(R.id.online_guide)).setMovementMethod(LinkMovementMethod.getInstance());
+
+		boolean isTelevision = getPackageManager().hasSystemFeature(PackageManager.FEATURE_LEANBACK);

```

Comment:
Rather than checking for TV, perhaps we should check if the system supports touch screen?

Reply:
I mostly agree, but this is also making an assumption about the space on the screen. Some devices do not have the necessary screen area to display the hints and it will result in other parts of the UI going missing as I haven't figured out how to make it scrollable :/

Reply:
Fair enough